### PR TITLE
Add model registry repo

### PR DESCRIPTION
**Pull request by afflom**

### CODE REVIEW

**Comment by sabre1041** [comment]
@afflom there are no changes included in this request

**Comment by afflom** [comment]
oops, moving too fast. I pushed the changes.

File: peribolos.yaml [hunk 1]
```diff
@@ -28,3 +28,7 @@ orgs:
         default_branch: main
         description: UOR sample Python gRPC client
         has_projects: true
+      ai-model-registry:
```
Comment (sabre1041) [inline peribolos.yaml]: Can this be added as to preserve alphabetic order?

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: peribolos.yaml (hunk 1 above)
Comment (afflom) [reply peribolos.yaml]: Fixed

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**